pkg/metrics/proxy.go

```diff
@@ -126,7 +151,7 @@ func (m *Proxy) createLoggregatorEnvelope(
 }
 
 func (m *Proxy) createGaugeMap(k v1.ResourceName, v resource.Quantity) map[string]*loggregator_v2.GaugeValue {
-	var gauges = map[string]*loggregator_v2.GaugeValue{}
+	gauges := map[string]*loggregator_v2.GaugeValue{}
```

Comment:
Small suggestion, doubt it matters though: probably could replace this with something like `gauges := make(map[string]...)`?

Reply:
Eek - I think that was `gofumpt` (our formatter) automatically changing that - so it wasn't something we deliberately changed from the original code. I suppose make is better if you know the expected size to reduce a couple of allocations as the map grows.